packages/components/src/Combobox/Combobox.types.ts

```diff
@@ -12,3 +12,8 @@ export interface ComboboxTriggerButtonProps
     Pick<ButtonProps, "type" | "variation" | "icon" | "iconOnRight"> {}
 
 export type ComboboxTriggerChipProps = ComboboxTriggerProps;
+
+export interface ComboboxOption {
+  id: string | number;
```

Comment:
I had this as only string, but decided to open it up because it's pretty normal to use numbers as ids too and I can't think of a solid reason we'd not want to support that